app/core/src/components/authentication/EmailLoginForm.js

```diff
@@ -24,18 +24,18 @@ export default class SimpleLoginForm extends React.Component<Props, State> {
     loading: false,
   };
 
-  _handleEmailChange = text =>
+  handleEmailChange = (text: string) =>
     this.setState({
       username: text,
     });
 
-  _handlePasswordChange = text =>
+  handlePasswordChange = (text: string) =>
```

Comment:
I didn't know about this. Flow doesn't care about underscored methods. This is not what we want right? 💩 🙂